strr-examiner-web/app/pages/registration/[registrationId].vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+const { t } = useI18n()
+const route = useRoute()
+const { manageAction } = useExaminerActions()
+const { updateRouteAndButtons } = useExaminerRoute()
+const { updateRegistrationStatus, getRegistrationById } = useExaminerStore()
+
+useHead({
+  title: t('page.dashboardList.title')
+})
+
+definePageMeta({
+  layout: 'examine',
+  middleware: ['auth']
+})
+
+const initialMount = ref(true)
+
+const { data: registration, status, error, refresh } = await useLazyAsyncData<
+  HousRegistrationResponse | undefined, ApplicationError
+>(
+  'registration-details-view',
+  async () => {
+    // slug will be there, otherwise the route will not be rendered and redirected to dashboard
+    const slug = Number(route.params.registrationId)
```

Comment:
No need to cast to Number here, because in getRegistrationById() it is converted back to string. It will also have an error if 'registrationId` in route params is not a number, unless it is handled somewhere else.

Reply:
The `route.params.registrationId` is giving a string where as `getRegistrationById` in examiner store expects a number. The `/registrations/{registration_id}`, registration_id needs to be an int otherwise it errors out.

Reply:
You can update the signature of the function and accept the string instead.

Reply:
Done